### PR TITLE
400 error Handling

### PR DESCRIPTION
**Pull request by udaydenduluri33**
400 error is now handled

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3467

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by kuda1992 — APPROVED** [review]

File: src/work-allocation/containers/available-tasks/available-tasks.component.ts [hunk 1]
```diff
@@ -113,9 +113,9 @@ export class AvailableTasksComponent extends TaskListWrapperComponent {
         type: InfoMessageType.WARNING,
         message: InfoMessage.TASK_NO_LONGER_AVAILABLE,
       });
-      if (handledStatus === 400) {
-        this.refreshTasks();
-      }
+    }
+    if (handledStatus === 400) {
```
Comment (paul-graham) [inline src/work-allocation/containers/available-tasks/available-tasks.component.ts:117]: What does this change accomplish? `handleStatus === 400` can only be true if the condition above it is also true (`handleStatus > 0`) so this simply adds another check that is guaranteed to evaluate to false if the first one is false. Or am I missing something? 🤷

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: src/work-allocation/utils/work-allocation-utils.ts [hunk 2]
```diff
@@ -45,6 +45,9 @@ export const handleFatalErrors = (status: number, navigator: Navigator, fatals?:
     case 503:
       navigator.navigate([ REDIRECTS.ServiceDown ]);
       return 0; // 0 indicates it has been handled.
+    case 400:
+      navigator.navigate([ REDIRECTS.ServiceDown ]);
+      return 400;
     default:
       // If it's anything other than a 401, 403, 500, or 503, we should not
```
Comment (paul-graham) [inline src/work-allocation/utils/work-allocation-utils.ts:52]: This comment is now out of date because you're also redirecting on a `400`.

**Review by paul-graham — CHANGES_REQUESTED** [review]

File: src/work-allocation/utils/work-allocation-utils.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/work-allocation/utils/work-allocation-utils.ts]: Yeah agreed. updated the comment now

File: src/work-allocation/containers/available-tasks/available-tasks.component.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/work-allocation/containers/available-tasks/available-tasks.component.ts]: Agreed. Done now.

**Review by paul-graham — APPROVED** [review]
All looks good. Thank you for making the amendments. 👍

**Pull request merged**